src/main/java/sungshin/sooon/controller/BoardController.java

```diff
@@ -0,0 +1,66 @@
+package sungshin.sooon.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Sort;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.data.domain.Pageable;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import sungshin.sooon.dto.PostRequestDto;
+import sungshin.sooon.dto.PostResponseDto;
+import sungshin.sooon.dto.PostUpdateRequestDto;
+import sungshin.sooon.model.Post;
+import sungshin.sooon.service.BoardService;
+
+import java.security.Principal;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/board")
+public class BoardController {
+
+    private final BoardService boardService;
+
+    // 전체 글 리스트
+    @GetMapping("/list")
+    public Page<Post> list(@PageableDefault(page = 1, sort = "created_at", direction = Sort.Direction.DESC) Pageable pageable, Model model) {
+        Page<Post> postList = boardService.getPostList(pageable);
+        model.addAttribute("postList", postList);
+
+        return boardService.getPostList(pageable);
+    }
+
+    // 게시글 저장
+    @PostMapping("/post")
```

Comment:
[image](https://s3.us-west-2.amazonaws.com/secure.notion-static.com/70da9b9f-2038-4185-8c80-edc43e220079/Untitled.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAT73L2G45O3KS52Y5%2F20210723%2Fus-west-2%2Fs3%2Faws4_request&X-Amz-Date=20210723T071602Z&X-Amz-Expires=86400&X-Amz-Signature=00cdd9883b0b2707958505dd450f93c214167559f4d655ea12cdb699a8c8ca16&X-Amz-SignedHeaders=host&response-content-disposition=filename%20%3D%22Untitled.png%22)

```suggestion
    @PostMapping("/")
```